partiql-tests-data/eval/query/join/joins.ion

```diff
@@ -593,7 +593,7 @@ join::[
         {
```

Comment:
This test logically corresponds to something like (I didn't rewrite the query, just added SQL tables):

```
CREATE TABLE t2 (
    name varchar(255),
    n int
);

CREATE TABLE t3 (
    name varchar(255),
    n int
);

INSERT INTO t2 (name, n) VALUES 
	('bb', 12), ('cc', 22), ('ee', 42);
INSERT INTO t3 (name, n) VALUES 
	('bb', 13), ('cc', 23), ('dd', 33);

SELECT * 
FROM 
		(SELECT name, n as s2_n, 2 as s2_2 FROM t2) as s2 
    LEFT JOIN 
	    (SELECT name, n as s3_n, 3 as s3_2 FROM t3) s3 ON s2.name = s3.name
```


And here we assert the result is 
```
<< 
    {'name': 'bb', 'n': 12, 'name':'bb', 'b':13},
    {'name': 'cc', 'n': 22, 'name':'cc', 'b':23},
    {'name': 'ee', 'n': 42, _2:null},
>>
```


The SQL-equivalent result might be encoded as 
```
<< 
    {'name': 'bb', 'n': 12, 'name':'bb', 'b':13},
    {'name': 'cc', 'n': 22, 'name':'cc', 'b':23},
    {'name': 'ee', 'n': 42, 'name':null, 'n':null},
>>
```

Reply:
Exactly. Here's a corresponding [DB Fiddle](https://www.db-fiddle.com/f/dYZcyMk7tEJjABmXY3Thts/1).